rpc/src/views.rs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use libra_types::account_config::AccountResource;
+use libra_types::byte_array::ByteArray;
+use serde::Serialize;
+
+#[derive(Serialize)]
+pub struct AccountView {
+    balance: u64,
```

Comment:
in javascript all numbers are "64 bit floats" ,  which maybe an issue if our number is large enough.  We need to either serialize numbers into strings or some other form (ethereum use BigInts)